src/EpiceaBrowsers/EpLogBrowserPresenter.class.st

```diff
@@ -324,11 +324,10 @@ EpLogBrowserPresenter >> eventsMenuActions [
 { #category : #'menu - operations' }
 EpLogBrowserPresenter >> fileOutSelection [
 
-	| entries outputDirectory |
+	| entries |
 	entries := self selectedEntryItems collect: [ :each | each entry ].
-	outputDirectory := log store directory.
 	
-	EpOmbuExporter askFileNameAndFileOut: entries in: outputDirectory.
+	EpOmbuExporter askFileNameAndFileOut: entries in: FileLocator imageDirectory.
```

Comment:
Why image directory and not working directory? In most cases it will be the same, but if the image is deployed in a separate directory, let's say read-only, working directory is better (and more user friendly IMO).